benchmark/rust/README.md

```diff
@@ -0,0 +1,45 @@
+# Benchmarking Rust implementation of lisk-db library
+
+## Prerequisites
+
+To be able to successfully run benchmarks, following applications need to be installed on a system:
+
+* [Valgrind](https://valgrind.org)
+* [KCachegrind](https://kcachegrind.sourceforge.net/html/Home.html)
+
+## List of all benchmark applications
+
```

Comment:
I guess it would be better to mentioned that the toolchain of rust should change to `1.59...`

Reply:
Done.